pkg/cimfs/cim_writer_windows.go

```
@@ -210,15 +291,14 @@ func (c *CimFsWriter) Close() error {
 	if err := c.commit(); err != nil {
 		return &OpError{Cim: c.name, Op: "commit", Err: err}
 	}
-	if err := winapi.CimCloseImage(c.handle); err != nil {
-		return &OpError{Cim: c.name, Op: "close", Err: err}
-	}
+	winapi.CimCloseImage(c.handle)
```

Why do we ignore errors now?